test/system/500-networking.bats

```diff
@@ -168,6 +168,7 @@ load helpers
     cid="$output"
 
     # check that dns is working inside the container
+    run_podman exec $cid cat /etc/resolv.conf    # FIXME: debugging for #11871
```

Comment:
This also needs `podman unshare --rootless-cni cat /etc/resolv.conf` (make sure to only run this as rootless user)
and the output of `ps uxww`

Reply:
Done, thanks